packages/ethers5/src/client.ts

```diff
@@ -313,7 +313,7 @@ export class Web3Modal extends Web3ModalScaffold {
   }
 
   public getAddress() {
-    return ethers.utils.getAddress(EthersStoreUtil.state.address as string)
+    return utils.getAddress(EthersStoreUtil.state.address as string)
```

Comment:
if `as string` is needed due to undefined being a possibility, check for that case explicitly. Otherwise ok 👍